ydb/library/yql/sql/sql.cpp

```diff
@@ -78,6 +78,11 @@ namespace NSQLTranslation {
             *actualSyntaxVersion = parsedSettings.SyntaxVersion;
         }
 
+        google::protobuf::Arena arena;
```

Comment:
But if we delete arena here after this function's execution, all returned proto messages will turn into a pumkin. It will be an error. Did you run it under ASAN with the case when `!parsedSettings.Arena`?